frontend/src/Pages/AddOpportunityForm/AddOpportunityForm.js

```diff
@@ -0,0 +1,151 @@
+
+import './AddOpportunityForm.css';
+import NavBar from '../../Components/NavBar/NavBar'
+import React, {useState} from 'react';
+import Footer from '../../Components/Footer/Footer';
+import SubmitButton from '../../Components/SubmitButton/SubmitButton'
+import Dropzone from 'react-dropzone';
+import request from 'superagent';
+import ImageUpload from '../../Components/ImageUpload/ImageUpload'
```

Comment:
For some reason, I am getting Module not found: for Dropzone, superagent, and image upload :( Did you make sure to commit all of your packages?

Reply:
I was able to get rid of react dropzone and superagent, realized that they are no longer relevant for this page (I can recommit without those two imports). The ImageUpload is a component that I added so it should be working if that component is implemented in the components folder. If you want it formatted a different way let me know and I can put the image upload content in a different folder if needed.